2023-04-10 12:08:28,570 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:10:37,476 django.request WARNING  Not Found: /favicon.ico
2023-04-10 12:12:10,855 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/hack/settings/base.py changed, reloading.
2023-04-10 12:12:11,829 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:12:16,341 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/hack/settings/base.py changed, reloading.
2023-04-10 12:20:47,900 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:21:01,214 django.request WARNING  Not Found: /favicon.ico
2023-04-10 12:22:48,887 django.request WARNING  Not Found: /favicon.ico
2023-04-10 12:26:08,111 django.request WARNING  Not Found: /favicon.ico
2023-04-10 12:27:46,567 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/views.py changed, reloading.
2023-04-10 12:27:51,646 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:27:58,147 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/views.py changed, reloading.
2023-04-10 12:27:59,096 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:03,538 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/views.py changed, reloading.
2023-04-10 12:28:04,420 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:06,794 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/views.py changed, reloading.
2023-04-10 12:28:07,646 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:12,063 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/views.py changed, reloading.
2023-04-10 12:28:12,999 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:21,551 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:22,395 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:23,689 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:24,695 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:29,022 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:30,249 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:32,638 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:33,552 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:38,899 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:39,753 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:42,109 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:43,000 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:49,385 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:50,150 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:28:56,494 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:28:57,266 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:29:00,458 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-10 12:29:01,451 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:29:13,112 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:29:13,115 django.request WARNING  Not Found: /assets/js/main.js
2023-04-10 12:29:13,123 django.request WARNING  Not Found: /assets/css/styles.css
2023-04-10 12:29:13,130 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:29:23,304 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:35:55,252 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:35:55,431 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:36:03,220 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:37:36,328 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:37:36,401 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:37:49,947 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:41:42,213 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:42:20,607 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:42:20,612 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:48:20,831 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-10 12:48:55,283 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:48:55,288 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:48:58,172 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:49:13,383 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:49:13,434 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:49:20,316 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:50:45,784 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:50:45,789 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:51:04,172 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:52:23,848 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:52:23,873 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:52:30,127 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:54:30,338 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:54:30,377 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:54:45,563 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:56:48,316 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:56:48,379 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:56:55,905 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 12:58:17,791 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 12:58:17,794 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 12:58:33,895 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 13:02:12,619 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 13:02:12,873 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 13:02:33,981 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 13:11:26,740 django.request WARNING  Not Found: /assets/js/scrollreveal.min.js
2023-04-10 13:11:26,838 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 13:12:10,984 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-10 13:18:02,085 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-10 13:18:13,418 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-11 22:14:46,391 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:15:11,013 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:15:13,158 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:15:29,704 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:15:30,696 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:15:34,065 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:15:35,161 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:15:40,519 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:15:41,418 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:15:43,694 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:15:44,463 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:05,991 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:06,770 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:08,026 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:08,747 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:14,972 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:15,769 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:19,123 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:19,987 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:22,281 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:23,239 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:25,576 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:26,375 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:27,750 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:28,554 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:29,932 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:30,837 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:32,298 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:33,239 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:37,639 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:38,531 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:39,849 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:40,559 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:44,884 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:45,818 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:48,182 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:49,055 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:16:50,370 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/urls.py changed, reloading.
2023-04-11 22:16:51,432 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:18:18,653 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-11 22:18:28,786 django.request WARNING  Not Found: /favicon.ico
2023-04-12 07:01:11,231 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 07:01:29,090 django.request WARNING  Not Found: /favicon.ico
2023-04-12 07:01:32,709 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:01:33,599 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:12:28,591 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:14:49,267 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:14:49,270 django.request WARNING  Not Found: /assets/img/body.jpg
2023-04-12 07:14:50,334 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:16:45,913 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:16:45,937 django.request WARNING  Not Found: /assets/img/body.jpg
2023-04-12 07:16:47,027 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:19:28,446 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:19:28,454 django.request WARNING  Not Found: /assets/img/body.jpg
2023-04-12 07:19:29,335 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:25:56,104 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:25:57,668 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:26:40,652 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:26:43,129 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:26:56,552 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:26:57,539 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:27:10,639 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 07:27:11,527 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 07:52:57,909 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 07:59:34,177 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/manage.py changed, reloading.
2023-04-12 07:59:35,959 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 08:03:03,068 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 08:05:23,647 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 08:07:11,234 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 08:07:38,626 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/hack/settings/base.py changed, reloading.
2023-04-12 08:07:40,297 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 08:55:17,481 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 09:05:41,590 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 10:42:15,120 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 10:42:28,571 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 10:42:43,906 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 10:47:33,157 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 10:47:35,497 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 10:47:38,007 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 10:47:39,265 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 11:25:20,973 django.security.DisallowedHost ERROR    Invalid HTTP_HOST header: 'fbfa-105-112-130-186.ngrok-free.app'. You may need to add 'fbfa-105-112-130-186.ngrok-free.app' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/http/request.py", line 167, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'fbfa-105-112-130-186.ngrok-free.app'. You may need to add 'fbfa-105-112-130-186.ngrok-free.app' to ALLOWED_HOSTS.
2023-04-12 11:25:22,905 django.request WARNING  Bad Request: /
2023-04-12 11:25:31,084 django.security.DisallowedHost ERROR    Invalid HTTP_HOST header: 'fbfa-105-112-130-186.ngrok-free.app'. You may need to add 'fbfa-105-112-130-186.ngrok-free.app' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/http/request.py", line 167, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'fbfa-105-112-130-186.ngrok-free.app'. You may need to add 'fbfa-105-112-130-186.ngrok-free.app' to ALLOWED_HOSTS.
2023-04-12 11:25:31,145 django.request WARNING  Bad Request: /
2023-04-12 11:25:58,892 django.security.DisallowedHost ERROR    Invalid HTTP_HOST header: 'fbfa-105-112-130-186.ngrok-free.app'. You may need to add 'fbfa-105-112-130-186.ngrok-free.app' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/utils/deprecation.py", line 133, in __call__
    response = self.process_request(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/http/request.py", line 167, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'fbfa-105-112-130-186.ngrok-free.app'. You may need to add 'fbfa-105-112-130-186.ngrok-free.app' to ALLOWED_HOSTS.
2023-04-12 11:25:58,959 django.request WARNING  Bad Request: /favicon.ico
2023-04-12 11:26:42,025 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 11:26:53,350 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 11:27:21,486 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 11:32:44,224 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 11:34:19,573 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 11:34:23,040 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 11:40:16,369 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 11:40:28,473 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 11:49:22,475 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 11:49:23,326 django.request WARNING  Not Found: /favicon.ico
2023-04-12 11:49:28,520 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 11:49:31,835 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 11:49:37,253 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 12:09:01,277 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:09:36,638 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:09:41,604 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:09:42,613 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:09:45,964 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:09:46,919 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:09:49,289 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:09:50,027 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:09:55,419 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:09:56,384 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:09:59,733 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:00,493 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:02,017 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:02,989 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:04,390 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:05,227 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:07,547 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:08,578 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:10,010 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:10,845 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:17,373 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:18,220 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:19,631 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:20,679 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:43,378 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:44,275 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:45,725 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:46,729 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:10:51,244 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:10:52,077 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:11:06,607 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:11:07,365 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:11:50,519 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:11:51,337 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:11:59,912 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:12:00,857 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:12:02,207 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:12:03,133 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:12:05,582 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:12:06,374 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:12:11,815 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:12:12,771 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:12:15,286 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:12:16,153 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:12:18,570 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:12:19,312 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:12:23,684 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:12:24,406 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:13:57,367 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 12:14:03,094 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 12:15:56,117 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:15:57,027 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:15:59,441 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:16:00,479 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:01,931 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:16:02,913 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:13,520 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:16:14,484 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:19,932 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:16:20,882 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:23,293 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:16:24,154 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:35,826 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:16:36,719 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:40,095 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:16:41,025 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:45,406 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:16:46,216 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:52,657 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:16:53,542 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:16:58,992 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:16:59,904 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:17:03,353 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:17:04,340 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:17:08,789 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:17:09,603 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:17:21,146 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:17:21,939 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:19:29,828 django.request WARNING  Not Found: /details
2023-04-12 12:19:31,185 django.request WARNING  Not Found: /favicon.ico
2023-04-12 12:19:47,170 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/urls.py changed, reloading.
2023-04-12 12:19:54,159 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:19:56,241 django.request ERROR    Internal Server Error: /posts
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/web/core/views.py", line 17, in details_page
    return render(request,'core/')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 23, in get_template
    contents = self.get_contents(origin)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 26, in get_contents
    return origin.loader.get_contents(origin)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/filesystem.py", line 22, in get_contents
    with open(origin.name, encoding=self.engine.file_charset) as fp:
IsADirectoryError: [Errno 21] Is a directory: '/home/madvirus/Documents/ai_hackathon/web/core/templates/core'
2023-04-12 12:20:14,267 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:20:15,312 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:20:40,167 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:20:40,995 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:20:46,122 django.request WARNING  Not Found: /favicon.ico
2023-04-12 12:22:48,891 django.request ERROR    Internal Server Error: /posts
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'obj.question'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/web/core/views.py", line 17, in details_page
    return render(request,'core/details.html')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 194: 'obj.question', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
2023-04-12 12:22:50,327 django.request WARNING  Not Found: /favicon.ico
2023-04-12 12:23:28,134 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 12:23:32,326 django.request ERROR    Internal Server Error: /posts
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'obj.question'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/web/core/views.py", line 17, in details_page
    return render(request,'core/details.html')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 194: 'obj.question', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
2023-04-12 12:23:33,217 django.request WARNING  Not Found: /favicon.ico
2023-04-12 12:24:58,028 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 12:25:08,822 django.request ERROR    Internal Server Error: /posts
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'obj.question'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/web/core/views.py", line 17, in details_page
    return render(request,'core/details.html')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 194: 'obj.question', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
2023-04-12 12:25:09,334 django.request WARNING  Not Found: /favicon.ico
2023-04-12 12:25:52,917 django.request ERROR    Internal Server Error: /posts
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'obj.question'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/web/core/views.py", line 17, in details_page
    return render(request,'core/details.html')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 194: 'obj.question', expected 'empty' or 'endfor'. Did you forget to register or load this tag?
2023-04-12 12:25:53,764 django.request WARNING  Not Found: /favicon.ico
2023-04-12 12:31:03,911 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 12:31:06,500 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 12:43:49,579 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 12:43:50,251 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-12 12:44:47,825 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-12 12:50:50,954 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:55:57,629 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:57:51,250 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:57:52,398 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:57:53,866 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:57:55,030 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:57:56,541 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-12 12:57:57,687 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 12:58:54,799 django.request WARNING  Not Found: /favicon.ico
2023-04-12 13:03:31,619 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/models.py changed, reloading.
2023-04-12 13:03:32,420 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-12 13:04:29,477 django.request WARNING  Not Found: /favicon.ico
2023-04-13 11:10:31,862 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-13 11:10:41,639 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-13 11:10:42,937 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-13 11:12:29,616 django.request WARNING  Not Found: /assets/js/anon.js
2023-04-13 11:12:30,340 django.request WARNING  Not Found: /assets/img/favicon.png
2023-04-13 11:57:06,094 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-04-13 11:57:41,386 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-13 12:00:35,269 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 10:29:36,359 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 10:37:16,405 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 10:37:27,713 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-02 10:37:29,906 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-02 10:39:22,799 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-02 10:39:23,497 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-02 13:02:08,861 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:03:26,861 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:03:28,339 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:03:34,752 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:03:35,493 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:03:45,377 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:03:58,807 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:03:59,852 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:04:02,216 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:04:03,062 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:04:07,399 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:04:08,271 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:04:10,602 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:04:11,513 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:04:15,935 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:04:16,963 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:04:19,305 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:04:20,141 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:04:53,158 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:04:53,949 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:04:58,367 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/web/core/views.py changed, reloading.
2023-05-02 13:04:59,106 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:09:33,767 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:09:56,993 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-02 13:11:13,070 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-05 12:44:18,899 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-05 12:44:44,894 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/pipelines.py changed, reloading.
2023-05-05 12:44:46,061 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-05 12:44:59,842 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-05 12:45:03,970 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-05 12:45:06,190 django.request WARNING  Not Found: /detail
2023-05-05 12:45:06,625 django.request WARNING  Not Found: /favicon.ico
2023-05-05 12:45:27,509 django.request WARNING  Not Found: /js/popper.min.js
2023-05-05 12:45:27,519 django.request WARNING  Not Found: /js/jquery.min.js
2023-05-05 12:45:27,554 django.request WARNING  Not Found: /js/bootstrap.min.js
2023-05-05 12:45:27,581 django.request WARNING  Not Found: /js/slick.min.js
2023-05-05 12:45:27,662 django.request WARNING  Not Found: /js/jquery.sticky-sidebar.min.js
2023-05-05 12:45:27,682 django.request WARNING  Not Found: /js/custom.js
2023-05-05 12:45:27,734 django.request WARNING  Not Found: /images/logo.svg
2023-05-05 12:45:27,766 django.request WARNING  Not Found: /images/other/about.jpg
2023-05-05 12:45:27,848 django.request WARNING  Not Found: /images/posts/tabs-1.jpg
2023-05-05 12:45:27,850 django.request WARNING  Not Found: /images/wave.svg
2023-05-05 12:45:27,942 django.request WARNING  Not Found: /images/posts/tabs-2.jpg
2023-05-05 12:45:27,946 django.request WARNING  Not Found: /images/posts/tabs-3.jpg
2023-05-05 12:45:28,024 django.request WARNING  Not Found: /images/widgets/widget-carousel-1.jpg
2023-05-05 12:45:28,029 django.request WARNING  Not Found: /images/widgets/widget-carousel-2.jpg
2023-05-05 12:45:28,118 django.request WARNING  Not Found: /images/ads/ad-360.png
2023-05-05 12:45:28,129 django.request WARNING  Not Found: /images/insta/insta-1.jpg
2023-05-05 12:45:28,242 django.request WARNING  Not Found: /images/insta/insta-2.jpg
2023-05-05 12:45:28,252 django.request WARNING  Not Found: /images/insta/insta-3.jpg
2023-05-05 12:45:28,286 django.request WARNING  Not Found: /images/insta/insta-4.jpg
2023-05-05 12:45:28,363 django.request WARNING  Not Found: /images/insta/insta-5.jpg
2023-05-05 12:45:28,390 django.request WARNING  Not Found: /images/insta/insta-6.jpg
2023-05-05 12:45:33,310 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:46:19,706 django.request WARNING  Not Found: /images/logo.svg
2023-05-05 12:46:19,710 django.request WARNING  Not Found: /images/other/about.jpg
2023-05-05 12:46:20,639 django.request WARNING  Not Found: /images/wave.svg
2023-05-05 12:46:20,680 django.request WARNING  Not Found: /images/posts/tabs-1.jpg
2023-05-05 12:46:20,724 django.request WARNING  Not Found: /images/posts/tabs-2.jpg
2023-05-05 12:46:20,811 django.request WARNING  Not Found: /images/posts/tabs-3.jpg
2023-05-05 12:46:20,815 django.request WARNING  Not Found: /images/widgets/widget-carousel-1.jpg
2023-05-05 12:46:20,914 django.request WARNING  Not Found: /images/widgets/widget-carousel-2.jpg
2023-05-05 12:46:20,942 django.request WARNING  Not Found: /images/ads/ad-360.png
2023-05-05 12:46:20,970 django.request WARNING  Not Found: /images/insta/insta-1.jpg
2023-05-05 12:46:21,031 django.request WARNING  Not Found: /images/insta/insta-2.jpg
2023-05-05 12:46:21,083 django.request WARNING  Not Found: /images/insta/insta-3.jpg
2023-05-05 12:46:21,110 django.request WARNING  Not Found: /images/insta/insta-4.jpg
2023-05-05 12:46:21,214 django.request WARNING  Not Found: /images/insta/insta-5.jpg
2023-05-05 12:46:21,253 django.request WARNING  Not Found: /images/insta/insta-6.jpg
2023-05-05 12:46:21,815 django.request WARNING  Not Found: /images/logo.svg
2023-05-05 12:46:21,947 django.request WARNING  Not Found: /images/map-bg.png
2023-05-05 12:46:22,108 django.request WARNING  Not Found: /images/widgets/widget-carousel-1.jpg
2023-05-05 12:46:25,863 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:47:52,371 django.request WARNING  Not Found: /images/logo.svg
2023-05-05 12:47:52,379 django.request WARNING  Not Found: /images/other/about.jpg
2023-05-05 12:47:53,147 django.request WARNING  Not Found: /images/insta/insta-1.jpg
2023-05-05 12:47:53,166 django.request WARNING  Not Found: /images/insta/insta-2.jpg
2023-05-05 12:47:53,249 django.request WARNING  Not Found: /images/insta/insta-3.jpg
2023-05-05 12:47:53,288 django.request WARNING  Not Found: /images/insta/insta-6.jpg
2023-05-05 12:47:53,313 django.request WARNING  Not Found: /images/insta/insta-5.jpg
2023-05-05 12:47:53,321 django.request WARNING  Not Found: /images/insta/insta-4.jpg
2023-05-05 12:47:53,452 django.request WARNING  Not Found: /images/logo.svg
2023-05-05 12:47:55,190 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:48:37,207 django.request WARNING  Not Found: /images/other/about.jpg
2023-05-05 12:48:37,216 django.request WARNING  Not Found: /images/insta/insta-1.jpg
2023-05-05 12:48:37,245 django.request WARNING  Not Found: /images/insta/insta-2.jpg
2023-05-05 12:48:37,336 django.request WARNING  Not Found: /images/insta/insta-3.jpg
2023-05-05 12:48:37,430 django.request WARNING  Not Found: /images/insta/insta-4.jpg
2023-05-05 12:48:37,477 django.request WARNING  Not Found: /images/insta/insta-5.jpg
2023-05-05 12:48:37,560 django.request WARNING  Not Found: /images/insta/insta-6.jpg
2023-05-05 12:48:37,646 django.request WARNING  Not Found: /images/logo.svg
2023-05-05 12:48:39,492 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:49:29,320 django.request WARNING  Not Found: /images/other/about.jpg
2023-05-05 12:49:31,487 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:50:37,793 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:51:04,210 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-05 12:51:05,996 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-05 12:52:44,599 django.request ERROR    Internal Server Error: /posts
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 54, in details_page
    return render(request,'core/more.html',context)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
2023-05-05 12:52:47,127 django.request WARNING  Not Found: /favicon.ico
2023-05-05 12:53:09,906 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:53:52,501 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:54:42,416 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:56:42,255 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:57:06,513 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 12:59:45,734 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 13:00:27,910 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 13:00:50,102 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 13:01:13,017 django.request WARNING  Not Found: /images/favicon.png
2023-05-05 13:01:38,201 django.request WARNING  Not Found: /images/favicon.png
2023-05-07 09:43:05,271 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-07 09:43:12,466 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-07 09:44:02,418 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-07 09:44:13,466 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-07 09:44:20,468 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-07 09:45:30,255 django.utils.autoreload INFO     /home/g1f7/Documents/ai_hackathon/ai_hackathon/core/views.py changed, reloading.
2023-05-07 09:45:30,774 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-07 09:45:36,868 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-07 15:26:50,265 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-07 15:30:08,120 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-07 15:30:09,612 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-07 15:30:46,131 django.request ERROR    Internal Server Error: /
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 17, in index
    response = np(str(passage))
  File "/home/madvirus/Documents/ai_hackathon/core/pipelines.py", line 176, in __call__
    return super().__call__(inputs)
  File "/home/madvirus/Documents/ai_hackathon/core/pipelines.py", line 51, in __call__
    sents, answers = self._extract_answers(inputs)
  File "/home/madvirus/Documents/ai_hackathon/core/pipelines.py", line 81, in _extract_answers
    sents, inputs = self._prepare_inputs_for_ans_extraction(context)
  File "/home/madvirus/Documents/ai_hackathon/core/pipelines.py", line 115, in _prepare_inputs_for_ans_extraction
    sents = sent_tokenize(text)
  File "/home/madvirus/.local/lib/python3.10/site-packages/nltk/tokenize/__init__.py", line 106, in sent_tokenize
    tokenizer = load(f"tokenizers/punkt/{language}.pickle")
  File "/home/madvirus/.local/lib/python3.10/site-packages/nltk/data.py", line 750, in load
    opened_resource = _open(resource_url)
  File "/home/madvirus/.local/lib/python3.10/site-packages/nltk/data.py", line 876, in _open
    return find(path_, path + [""]).open()
  File "/home/madvirus/.local/lib/python3.10/site-packages/nltk/data.py", line 583, in find
    raise LookupError(resource_not_found)
LookupError: 
**********************************************************************
  Resource [93mpunkt[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt/PY3/english.pickle[0m

  Searched in:
    - '/home/madvirus/nltk_data'
    - '/usr/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/share/nltk_data'
    - '/usr/local/share/nltk_data'
    - '/usr/lib/nltk_data'
    - '/usr/local/lib/nltk_data'
    - ''
**********************************************************************

2023-05-07 15:32:00,357 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-07 15:33:20,864 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-07 15:33:24,383 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-07 15:34:19,705 django.request WARNING  Not Found: /favicon.ico
2023-05-07 19:14:16,588 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-07 19:15:13,403 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-07 19:15:16,443 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-07 19:15:48,452 django.request WARNING  Not Found: /favicon.ico
2023-05-07 19:21:19,562 django.request ERROR    Internal Server Error: /
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 41, in index
    return render(request,'core/details.html',{"questions":questions, "answers":answers})
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/defaulttags.py", line 853, in do_for
    nodelist_loop = parser.parse(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 513, in parse
    raise self.error(token, e)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/defaulttags.py", line 839, in do_for
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'for' statements should use the format 'for x in y': for answer is answers
2023-05-08 06:35:33,588 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:38:22,072 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:39:26,704 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-08 06:39:32,780 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 06:40:30,343 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:40:32,317 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:40:33,071 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:40:33,808 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:40:35,320 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:40:36,108 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:40:36,818 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:40:37,576 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:42:03,177 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-08 06:42:06,366 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 06:42:16,269 django.request WARNING  Not Found: /user/update-profile/
2023-05-08 06:42:47,315 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 06:43:28,608 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:44:50,791 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 06:44:51,808 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:45:58,578 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 06:46:17,649 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:47:10,341 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/urls.py changed, reloading.
2023-05-08 06:47:16,354 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:47:36,104 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/urls.py changed, reloading.
2023-05-08 06:47:38,287 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:48:02,759 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/urls.py changed, reloading.
2023-05-08 06:48:04,916 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:48:36,676 django.request WARNING  Not Found: /assets/js/anon.js
2023-05-08 06:48:40,373 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 06:48:49,245 django.request ERROR    Internal Server Error: /process/
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 61, in processPassage
    return JsonResponse({"status":200})
NameError: name 'JsonResponse' is not defined
2023-05-08 06:49:17,984 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 06:49:28,425 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 06:55:24,624 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 06:57:24,388 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 06:59:13,535 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 06:59:51,619 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:01:24,534 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:07:53,707 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:08:33,456 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:11:01,875 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:12:14,371 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:13:32,046 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:16:23,797 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:17:13,159 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:18:23,684 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 07:18:30,928 django.request WARNING  Not Found: /path/to/redirect
2023-05-08 07:18:32,911 django.request WARNING  Not Found: /favicon.ico
2023-05-08 09:09:47,669 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 09:11:30,052 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 09:13:57,482 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 09:16:19,828 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 09:18:10,896 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 09:18:25,238 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 09:20:35,015 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 11:04:18,672 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 11:05:25,200 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 11:09:04,324 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 11:10:39,502 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 11:12:00,720 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 11:16:10,537 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 11:18:47,974 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 11:25:18,073 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 11:26:16,416 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 11:27:36,942 django.request WARNING  Not Found: /details
2023-05-08 11:27:38,719 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:28:32,486 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/urls.py changed, reloading.
2023-05-08 11:28:40,445 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 11:30:55,073 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 11:33:34,070 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 11:35:01,012 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:44:10,028 django.request ERROR    Internal Server Error: /details
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 53, in details_page
    return render(request,'core/details.html',context)
NameError: name 'context' is not defined
2023-05-08 11:44:11,549 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:44:24,167 django.utils.autoreload INFO     /home/madvirus/Documents/ai_hackathon/core/views.py changed, reloading.
2023-05-08 11:44:55,201 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-05-08 11:45:43,764 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:47:23,227 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:49:12,040 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:50:44,830 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:53:28,108 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:58:56,380 django.request WARNING  Not Found: /favicon.ico
2023-05-08 11:59:47,814 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:01:36,906 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:03:25,436 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:04:30,766 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:04:35,776 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:06:23,637 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:07:42,285 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:08:03,616 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 12:10:28,645 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:11:08,844 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:12:15,267 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 12:14:00,214 django.request ERROR    Internal Server Error: /details
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 53, in details_page
    return render(request,'core/details.html')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
2023-05-08 12:14:02,540 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:14:23,129 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:15:47,120 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:17:06,340 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:19:12,263 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:20:51,569 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:21:14,311 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:22:42,347 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:23:32,830 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:24:36,782 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:26:54,538 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:27:33,364 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:29:35,581 django.request WARNING  Not Found: /assets/img/favicon.png
2023-05-08 12:31:31,894 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:31:37,747 django.request ERROR    Internal Server Error: /details
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 53, in details_page
    return render(request,'core/details.html')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 54: 'endfor'. Did you forget to register or load this tag?
2023-05-08 12:31:39,858 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:32:18,141 django.request ERROR    Internal Server Error: /details
Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/madvirus/Documents/ai_hackathon/core/views.py", line 53, in details_page
    return render(request,'core/details.html')
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/madvirus/.local/lib/python3.10/site-packages/django/template/base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 54: 'endfor'. Did you forget to register or load this tag?
2023-05-08 12:32:19,736 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:33:35,162 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:34:16,980 django.request WARNING  Not Found: /favicon.ico
2023-05-08 12:34:55,700 django.request WARNING  Not Found: /assets/img/favicon.png
